docs/guides/creatingasplitter.md

```diff
@@ -1,17 +1,27 @@
-# Creating a Splitter
+# Creating an Event Splitter
 
-Splitter is a part of TriggerMesh routing solution that has a simple purpose to
-split JSON arrays into multiple CloudEvents.
+An event `Splitter` is part of the TriggerMesh routing solution. It has a simple purpose of splitting JSON arrays into multiple [CloudEvents](https://cloudevents.io/).
 
-## Prerequisites
+!!! Info "Prerequisites"
+    You need a working TriggerMesh platform installation. See the [installation steps](installation.md). You can verify that the API is available with the following command:
+    
+    ```console
+       $ kubectl get crd|grep splitter
+       splitters.routing.triggermesh.io                        2021-10-06T09:01:38Z
+    ```
 
-1. K8s cluster and configured kubectl
-1. [Knative Serving and Eventing Operators](https://knative.dev/docs/admin/install/knative-with-operators/)
-1. [TriggerMesh Function](https://github.com/triggermesh/function)
+![](../assets/images/splitter.png)
+
+Let's create all the required objects:
+
+- [x] The sockeye target which serves as an event display.
+- [x] The `PingSource` which produces a JSON array in its payload.
+- [x] The `Splitter` to generate the multiple events.
```

Comment:
Same comment as before, was this really meant as a TODO style list with checkboxes?

Reply:
it is. I am not in love with it, but I think we can change it in the coming days.